Views/Layout-EmailTemplate.cshtml

```diff
@@ -0,0 +1,45 @@
+@{
+    const string blockName = "emailTemplate";
+}
+
+<!DOCTYPE html>
+<html lang="@Orchard.CultureName()">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width" initial-scale="1">
+    <!--[if !mso]>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <![endif]-->

```

Comment:
What is this for?

Reply:
Not sure, but I saw this in multiple sample email template HTML. By reading it my guess is that it won't set this value if the email is being viewed in Microsoft Outlook otherwise it'd break it (since it uses IE 6 FYI hence the ugly emails in Outlook).